src/controller.ts

```diff
@@ -77,6 +67,23 @@ export function addController (
       } else {
         operation.responses![httpCode] = { description: description ?? '' }
       }
+      if (parseInt(httpCode, 10) >= 200 && parseInt(httpCode, 10) <= 299) hasSuccessResponse = true
```

Comment:
Not a big fan of one-liner, could you:
```suggestion
      if (parseInt(httpCode, 10) >= 200 && parseInt(httpCode, 10) <= 299) {
      	hasSuccessResponse = true
      }
```